### ISSUE TITLE
Breakout E2E tests in suites so we can run in parallel

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#9039]
Currently, our E2E runs in serial and takes over an hour to run. We need to break this down into separate suites.

For now, we can ignore coverage; we should circle back to that in a separate ticket.

**Comment by nwmac** [comment]
https://github.com/rancher/dashboard/issues/9064 is a duplicate - closing this one as the other has more information attached
